compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIETileAndFuse.cpp

```diff
@@ -125,6 +130,10 @@ void AMDAIETileAndFusePass::runOnOperation() {
   SmallVector<int64_t> tileSizesVal;
   if (auto loweringConfig = getLoweringConfig(consumerOp)) {
     tileSizesVal = loweringConfig.getTileSizeVals(tilingLevel);
+  } else if (isa<linalg::GenericOp>(consumerOp)) {
+    // TODO(vivian): Fix in upstream. For now this is a temp hack to get around
```

Comment:
I flagged this on the other PR as well.... But this has been festering for a while....  This is a pretty big hammer.... I will talk to Vivian today, and me or her can get around this.

Reply:
Yes, so the plan is we don't do this in upstream, right? I'll add a workaround inside this repo. Should I just use `getCompilationInfo/setCompilationInfo` and add them inside the packing pass?